src/components/reader-player/Player.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import Player from "./Player";
+
+const meta: Meta<typeof Player> = {
+  title: "ReaderPlayer / Player",
+  component: Player,
+  argTypes: {
+    identifier: {
+      control: { type: "text" }
+    },
+    orderId: {
+      control: { type: "text" }
+    }
+  }
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Player>;
+
+export const WithIdentifier: Story = {
+  render: () => <Player identifier="9788702366600" />
+};
+
+// Works only if the matrial is reserved
+export const WithOrderId: Story = {
+  render: () => <Player orderId="" />
```

Comment:
Could we pls somehow center it on the page so that it all doesn't get cut off in the story?
![image](https://github.com/user-attachments/assets/ead68680-6585-4e58-a360-eae7f53be818)

Reply:
Yes, that is not how it should look, but that’s not how it is on my storybook. 😅

<img width="1840" alt="Skærmbillede 2024-12-03 kl  11 13 09" src="https://github.com/user-attachments/assets/c57e1e7e-95d6-4bcc-ba67-31d658cabe41">